docs/release/trg-4/trg-4-06.md

```diff
@@ -68,50 +81,20 @@ Closing the notice, we provide a general statement about potentially contained o
 of our images. We use the following text:
 
 ```markdown
-As with all Docker images, these likely also contain other software which may be under other licenses 
+As with all Docker images, these likely also contain other software which may be under other licenses
 (such as Bash, etc. from the base distribution, along with any direct or indirect dependencies of the primary software being contained).
 
 As for any pre-built image usage, it is the image user's responsibility to ensure that any use of this image complies with any relevant licenses for all software contained within.
 ```
 
-## Best practice and examples
-
-- Use "Container images" as headline to link to the container image notice files from the repositories README.md file
-- Name the notice file "notice.md"
-- Provide important information about your image with your notice. Examples:
-  - Available volumes
-  - Default port the service is listening on
-
-### GitHub workflow step
-
-Keeping the DockerHub description up-to-date with the latest notice content is crucial.
-The following workflow step can be used as part of the DockerHub publishing workflow to ensure that:
-
-```yaml
-# DockerHub publishing worklflow
-
-...
-
-# https://github.com/peter-evans/dockerhub-description
-- name: Update Docker Hub description
-  # Ensure the DockerHub description is updated with contents from a PR (usually images are not published on PR)
-  if: github.event_name != 'pull_request'
-  uses: peter-evans/dockerhub-description@v3
-  with:
-    username: ${{ secrets.DOCKER_HUB_USER }}
-    password: ${{ secrets.DOCKER_HUB_TOKEN }}
-    repository: ${{ env.IMAGE_NAMESPACE }}/${{ env.IMAGE_NAME }}
-    # IMPORTANT: Adjust this to the actual path of your container image notice
-    readme-filepath: <path/to/your/notice.md>
-```
-
-### Examples
+## Examples
 
 The following examples are shown as reference, to see already existing and complete versions of a 'Notice for docker images'.
 They **can not** be used for your product without modifications.
 
-- [edc-controlplane-memory-hashicorp-vault](https://github.com/eclipse-tractusx/tractusx-edc/blob/main/edc-controlplane/edc-controlplane-memory-hashicorp-vault/notice.md)
-- [Country Risk](https://github.com/eclipse-tractusx/vas-country-risk/blob/main/DOCKER_NOTICE.md)
+Good example for notice integrated in toplevel `README.md`: [IRS](https://github.com/eclipse-tractusx/item-relationship-service#notice-for-docker-image)
+
+Good example for a dedicated notice file: [edc-controlplane-memory-hashicorp-vault](https://github.com/eclipse-tractusx/tractusx-edc/edit/main/edc-controlplane/edc-controlplane-memory-hashicorp-vault/notice.md)
 
 ## Already collected base image information
 
```

Comment:
Something that helped me was the information that the image later on running in the container is the base image under consideration, e.g. meaning that in case of multi-stage builds, the last one's the relevant. This would be a quality of live improvement. Opinions?

Reply:
You mean in a sense of adding a hint to the TRG?

Reply:
Exactly, just for better understandability for new joiners

Reply:
Hint added to the Why section